datasets/world_bank_intl_debt/series_summary/pipeline.yaml

```diff
@@ -0,0 +1,166 @@
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+resources:
+
+  - type: bigquery_table
+    table_id: series_summary
+
+    description: "Series Summary table"
+
+dag:
+  airflow_version: 1
+  initialize:
+    dag_id: series_summary
+    default_args:
+      owner: "Google"
+
+      depends_on_past: False
+      start_date: "2021-03-01"
+    max_active_runs: 1
+    schedule_interval: "@daily"
+    catchup: False
+    default_view: graph
+
+  tasks:
+    - operator: "KubernetesPodOperator"
+
+
+      description: "Run CSV transform within kubernetes pod"
+
+      args:
+
+        task_id: "series_summary_transform_csv"
+
+        startup_timeout_seconds: 600
+
+        name: "series_summary"
+
+        namespace: "default"
+
+        affinity:
+          nodeAffinity:
+            requiredDuringSchedulingIgnoredDuringExecution:
+              nodeSelectorTerms:
+                - matchExpressions:
+                    - key: cloud.google.com/gke-nodepool
+                      operator: In
+                      values:
+                        - "pool-e2-standard-4"
+
+
+        image_pull_policy: "Always"
+
+        image: "{{ var.json.world_bank_intl_debt.container_registry.run_csv_transform_kub }}"
+
+        env_vars:
+          SOURCE_URL: "gs://pdp-feeds-staging/RelayWorldBank/IDS_CSV/IDSSeries.csv"
+          SOURCE_FILE: "files/data.csv"
+          COLUMN_TO_REMOVE: "Unnamed: 20"
+          TARGET_FILE: "files/data_output.csv"
+          TARGET_GCS_BUCKET: "{{ var.value.composer_bucket }}"
+          TARGET_GCS_PATH: "data/world_bank_intl_debt/series_summary/data_output.csv"
+          PIPELINE_NAME: "series_summary"
+          CSV_HEADERS: >-
+            ["series_code" ,"topic" ,"indicator_name" ,"short_definition" ,"long_definition" ,"unit_of_measure" ,"periodicity" ,"base_period" ,"other_notes" ,"aggregation_method" ,"limitations_and_exceptions" ,"notes_from_original_source" ,"general_comments" ,"source" ,"statistical_concept_and_methodology" ,"development_relevance" ,"related_source_links" ,"other_web_links" ,"related_indicators" ,"license_type"]
+          RENAME_MAPPINGS: >-
+            {"Series Code":"series_code" ,"Topic":"topic" ,"Indicator Name":"indicator_name" ,"Short definition":"short_definition" ,"Long definition":"long_definition" ,"Unit of measure":"unit_of_measure" ,"Periodicity":"periodicity" ,"Base Period":"base_period" ,"Other notes":"other_notes" ,"Aggregation method":"aggregation_method" ,"Limitations and exceptions":"limitations_and_exceptions" ,"Notes from original source":"notes_from_original_source" ,"General comments":"general_comments" ,"Source":"source" ,"Statistical concept and methodology":"statistical_concept_and_methodology" ,"Development relevance":"development_relevance" ,"Related source links":"related_source_links" ,"Other web links":"other_web_links" ,"Related indicators":"related_indicators" ,"License Type":"license_type"}
```

Comment:
^

Reply:
As discussed, we are skipping this as of now